app/sample.yaml

```diff
@@ -16,3 +16,4 @@ tests:
       - nxp_adsp_imx8
       - nxp_adsp_imx8x
       - nxp_adsp_imx8m
+      - nxp_adsp_imx8ulp
```

Comment:
I suspect you can't do that without listing it in `platform_allow` too. Try `./scripts/twister -T ../modules/audio/sof/app/`

Strange, there should be a git conflict with #8429 merged 4 hours ago... GitHub is probably running behind, there were some outages today.

Reply:
There was no git conflict, the changes were far away enough from each other.